flow/src/main/java/com/sequenceiq/flow/service/flowlog/FlowLogDBService.java

```diff
@@ -94,10 +98,29 @@ public String getSerializedString(Object object) {
         } catch (Exception e) {
             LOGGER.debug("Somehow can not serialize object to string, try another method..", e);
             objectAsString = JsonUtil.writeValueAsStringSilent(object);
+            if (objectAsString == null) {
+                LOGGER.debug("Cannot serialize with JsonUtils, try with gson");
+                objectAsString = getGsonWithExceptionSerializer().toJson(object);
+            }
+        }
+        if (objectAsString != null) {
+            LOGGER.debug("Serialized string: {}", objectAsString);
+        } else {
+            try {
+                LOGGER.info("Object cold not be serialized: {}", object);
+            } catch (Exception e) {
+                LOGGER.warn("Cannot write the object into the log. Maybe it is a lazy init exception", e);
+            }
```

Comment:
would it be better to move this logic into `JsonUtil`?

Reply:
I have removed this logic